webSocket/client/client.go

```diff
@@ -120,47 +101,38 @@ func (c *client) Start() {
 			}
 		}
 	}()
+
+	go func() {
+		for {
+			_ = c.transceiver.Listen(c.wsConn)
+
+			select {
+			default:
+			case <-c.safeCloser.ChanClose():
```

Comment:
maybe `default` should be after `case`? please test

Reply:
fixed